apps/barista-design-system/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -54,6 +61,8 @@ export class BaIconColorWheel {
   /** @internal */
   @ViewChild('colorwheel', { static: true }) _colorWheel: ElementRef;
 
+  @ViewChild('symbol', { static: true }) _symbol: ElementRef;
```

Comment:
Does this work without the #named reference in the template?

Reply:
The delete button was too excited ^^

Reply:
Furthermore, I question why you need the view child it looks like that it is not used in your PR if I'm correct so please remove it or use it.